<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>carplate</class>
 <widget class="QWidget" name="carplate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1843</width>
    <height>1004</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>carplate</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>130</y>
     <width>1571</width>
     <height>911</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>531</width>
       <height>351</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>900</width>
       <height>500</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="icon.qrc">:/start.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget_carinfo">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>0</y>
       <width>581</width>
       <height>351</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>460</y>
       <width>395</width>
       <height>111</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>车牌号：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>Agency FB</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>颜色：</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="lineEdit_num">
          <property name="minimumSize">
           <size>
            <width>290</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>290</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>14</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_color">
          <property name="minimumSize">
           <size>
            <width>290</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>290</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>14</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>610</y>
       <width>421</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="button_open">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>进入</string>
        </property>
        <property name="icon">
         <iconset resource="icon.qrc">
          <normaloff>:/start.png</normaloff>:/start.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_close">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>关闭</string>
        </property>
        <property name="icon">
         <iconset resource="icon.qrc">
          <normaloff>:/close.png</normaloff>:/close.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>40</y>
       <width>211</width>
       <height>141</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_date">
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_time">
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>26</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 255, 0);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QComboBox" name="comboBox_selectcam">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>0</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_map">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>370</y>
       <width>821</width>
       <height>481</height>
      </rect>
     </property>
     <property name="text">
      <string>地图</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_point">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>250</y>
       <width>151</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>中心坐标</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_city">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>160</y>
       <width>181</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>城市</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QTableWidget" name="tableWidget_select">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>1271</width>
       <height>521</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_carnumselect">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>261</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>14</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="dateTimeEdit">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>50</y>
       <width>251</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>14</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="dateTimeEdit_2">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>50</y>
       <width>251</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>14</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>50</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>始</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>50</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>至</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_select">
     <property name="geometry">
      <rect>
       <x>1170</x>
       <y>50</y>
       <width>121</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>181</width>
     <height>831</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>操作</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="button_camerainfo">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>摄像头</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_table">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>查询信息</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_exit">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>761</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="text">
    <string>GGBond的停车管理系统</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
